ext/mysqli/tests/bug79375.phpt

```diff
@@ -2,6 +2,7 @@
 Bug #79375: mysqli_store_result does not report error from lock wait timeout
 --SKIPIF--
 <?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
```

Comment:
There is a corresponding test in pdo_mysql, might want to mark that one as well.

Reply:
Ah, good catch!